test/text/web_vtt_layout_integration.js

```diff
@@ -248,6 +248,23 @@ filterDescribe('WebVTT layout', shaka.test.TextLayoutTests.supported, () => {
       await helper.checkScreenshot('line-50');
     });
 
+    // Regression case for http://b/259121343
+    // The top center of the cue box should be 85% from the top and
+    // horizontally centered.
+    // NOTE: The native version is wrong on Chrome and Edge due to layout bugs
+    // in Chrome.  TODO: File crbug about this.
```

Comment:
Please, add the link before merge the PR

Reply:
It will take me a while to craft a bug report, and I want to move forward with Shaka releases to beat our deadline for CAF.

Reply:
I also just found a crbug about VTT settings that we filed in 2016, and is still unfixed.  So I'm not holding my breath on this new one.

Reply:
Ok! So, feel free to merge it

Reply:
I wound up being blocked on a lab issue to update the screenshots, so I took the time to create a simple repro case and file the Chrome bug.  https://crbug.com/1411464  I'll add that to the PR as soon as I get the updated screenshots.

Reply:
Thanks!